### PR TITLE
Skip network session affinity test cases on mce agent power

### PR DESCRIPTION
**Pull request by dharaneeshvrd**
Need to skip these till a fix is raised for respective bug https://issues.redhat.com/browse/OCPBUGS-12841

Add Shilpa-Gokul as owner

### CODE REVIEW

**Comment by dharaneeshvrd** [comment]
/pj-rehearse periodic-ci-openshift-hypershift-release-4.14-periodics-e2e-mce-power-conformance

File: ci-operator/step-registry/hypershift/mce/power/OWNERS [hunk 1]
```diff
@@ -1,13 +1,9 @@
...
 - dharaneeshvrd
+- Shilpa-Gokul
 options: {}
 reviewers:
-- csrwng
-- enxebre
-- sjenning
 - mkumatag
 - dharaneeshvrd
+- Shilpa-Gokul
```
Comment (mkumatag) [inline ci-operator/step-registry/hypershift/mce/power/OWNERS:9]: please add the newline in the end, applies in the other places as well

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: ci-operator/step-registry/hypershift/mce/power/OWNERS (hunk 1 above)
Comment (dharaneeshvrd) [reply ci-operator/step-registry/hypershift/mce/power/OWNERS]: fixed

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**